content/2.blog/2025/05/06/portfolio-redesign.md

```
@@ -0,0 +1,261 @@
+}
+```
+
+
+
+## 📊 ビフォーアフターの比較
+
+### デザインの自由度
+
+VitePressからの移行で最も大きく変わったのは、デザインの自由度でした：
+
+**Before (VitePress)**
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading
null

(MD036, no-emphasis-as-heading)